src/api/app/models/project.rb

```diff
@@ -968,6 +976,44 @@ def branch_to_repositories_from(project, pkg_to_enable, opts = {})
     end
   end
 
+  def branch_remote_repositories(project)
+    remote_project = RemoteProject.new(name: project)
```

Comment:
Please do not use `RemoteProject` for this. A `RemoteProject` is a `Project` defining the interconnect link. `project` will always be some `Project` on the remote instance below this. 

Speaking https://build-test.opensuse.org:
[openSUSE.org](https://build-test.opensuse.org/project/show/openSUSE.org) is a `RemoteProject`
[openSUSE.org:OBS:Server:Unstable](https://build.opensuse.org/project/show/OBS:Server:Unstable) is what you will get in the `project` variable.